pkg/platform/model/recipe.go

```diff
@@ -133,17 +138,20 @@ func commitToOrder(commitID strfmt.UUID, owner, project string, hostPlatform *st
 	return order, nil
 }
 
-func fetchRecipeID(commitID strfmt.UUID, owner, project string, hostPlatform *string) (*strfmt.UUID, *failures.Failure) {
+func fetchRecipeID(commitID strfmt.UUID, owner, project, orgID string, private bool, hostPlatform *string) (*strfmt.UUID, *failures.Failure) {
 	var err error
 	params := iop.NewSolveOrderParams()
 	params.Order, err = commitToOrder(commitID, owner, project, hostPlatform)
```

Comment:
We're still passing hostPlatform here, given the new additions for asserting the platform should this be removed?

Reply:
Originally `commitToOrder` was also filtering platforms from the returned order. It's no longer needed here, but is still needed in `fetchRawRecipes`, where it now lives until we address this: https://www.pivotaltracker.com/story/show/174076298.